src/main/java/org/opentripplanner/routing/api/request/preference/WalkPreferences.java

```diff
@@ -28,13 +28,16 @@ public final class WalkPreferences implements Serializable {
   private final double stairsTimeFactor;
   private final double safetyFactor;
 
+  private final double escalatorReluctance;
+
   private WalkPreferences() {
     this.speed = 1.33;
     this.reluctance = 2.0;
     this.boardCost = 60 * 10;
     this.stairsReluctance = 2.0;
     this.stairsTimeFactor = 3.0;
     this.safetyFactor = 1.0;
+    this.escalatorReluctance = 2.0;
```

Comment:
I think this value should be lower. Not quite sure what though. Kinda weird that the default for reluctance and stairsReluctance are the same. Maybe the stairsReluctance is added on top of the normal reluctance but I'm not sure.

Reply:
We investigated the stairsReluctance a bit in the dev meeting and the cost comes from both the stairsReluctance and the stairsTimeFactor so the default reluctance value is probably approriate as is.

For the escalatorReluctance, maybe set this default value as 1.5.